### PR TITLE
VODE90 for simplified SDC

### PR DESCRIPTION
**Pull request by zingale**
These changes let us use the VODE90 integrator for the Simplified-SDC

### CODE REVIEW

**Comment by zingale** [comment]
this is ready for review

tests do as they should

**Comment by dwillcox** [comment]
Since we're changing `USE_SDC` to `USE_SIMPLIFIED_SDC` can we also change the preprocessor definition from `SDC` to `SIMPLIFIED_SDC`?

That would clarify what's meant by:

```
#ifdef SDC
 ...
#elif TRUE_SDC
 ...
#else
 ...
#endif
```

Otherwise looks good!

**Comment by zingale** [comment]
I think this is ready for re-review

Note: because of this change, any application codes will need to switch their builds to set `-DSIMPLIFIED_SDC` instead of `-DSDC`

File: integration/VODE90/actual_integrator_simplified_sdc.F90 [hunk 1]
```diff
@@ -97,40 +60,46 @@ subroutine actual_integrator(state_in, state_out, dt, time)
...
     !
     ! **NOTE** if you reduce these tolerances, you probably will need
     ! to (a) decrease dT_crit, (b) increase the maximum number of
     ! steps allowed.
 
-    atol(SFS:SFS-1+nspec) = atol_spec ! mass fractions
-    atol(SEDEN)           = atol_enuc ! total energy
-    atol(SEINT)           = atol_enuc ! internal energy
+    dvode_state % atol(SFS:SFS-1+nspec) = atol_spec ! mass fractions
+    dvode_state % atol(SEDEN)           = atol_temp ! temperature
```
Comment (maxpkatz) [inline integration/VODE90/actual_integrator_simplified_sdc.F90]: Why does SEDEN hold temperature?

**Pull request merged**